homeassistant/components/apcupsd/sensor.py

```diff
@@ -97,8 +98,8 @@
     " Seconds": TIME_SECONDS,
     " Percent": UNIT_PERCENTAGE,
     " Volts": UNIT_VOLT,
-    " Ampere": "A",
-    " Volt-Ampere": "VA",
+    " Ampere": ELECTRICAL_CURRENT_AMPERE,
+    " Volt-Ampere": f"{UNIT_VOLT}{ELECTRICAL_CURRENT_AMPERE}",
```

Comment:
IMHO we should not combine it like this...

Reply:
So what should I do with this? Revert my change?

Reply:
Well, a unit of measurement is an identifier that represents the magnitude or quantity for the number shown.

In this case, the unit is volt-ampere (`VA`) which is the apparent power. While just `V` = voltage and just `A` = the electric change in ampere.

`VA` is a valid unit, the apparent power, which should have its own constant.

The reasoning for having centralized constants for a single unit of measurement is the ability to maybe, in the future, being able to convert between those. In the short term, they add consistency.

Reply:
Okay, I will add this costant to const.py as well and use it here ✌

Reply:
All done here AFAIK